server/models/EventInviteModel.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+const eventInviteSchema = new mongoose.Schema({
```

Comment:
Is the intent to use the event ID here to query for the event details in the eventSchema? (i.e. datetime, location, etc.)?